distribution/zip/jballerina/bin/bal

```diff
@@ -251,7 +251,7 @@ BALLERINA_CLASSPATH=$BALLERINA_CLASSPATH:$BALLERINA_CLASSPATH_EXT
 CMD_LINE_ARGS="-Xbootclasspath/a:"$BALLERINA_XBOOTCLASSPATH" \
                -Xms256m -Xmx1024m \
                -XX:+HeapDumpOnOutOfMemoryError \
-               -XX:HeapDumpPath="$(pwd)" \
+               -XX:HeapDumpPath="''" \
```

Comment:
Is this correct, In Windows this is empty and here it is not. 

Btw, this is an optional flag. If we want the default behavior, remove this flag. Fix this in `bal.bat` as well.

Reply:
@hasithaa I checked this locally. In Windows, it works when we give the path as `""`. But in Linux, it is working when we give `"''"`.

Reply:
Don't we have to remove the windows one as well?

Reply:
Yes. But have to check in a windows machine. By the time, I will remove from ubuntu since this causes an installer test failure.

Reply:
@hasithaa Removed the configuration in windows as well after testing in windows.